bin/feature-search.js

```diff
@@ -0,0 +1,153 @@
+#! /usr/env/bin node
+
+const fs = require( 'fs' );
+const chalk = require( 'chalk' );
+
+async function getConfigFeatures( config ) {
+	const file = `${ __dirname }/../config/${ config }.json`;
+	const json = await fs.readFileSync( file, 'utf8' );
+	const data = JSON.parse( json );
+
+	return data.features;
+}
+
+function calcLongestConfig( result ) {
+	let longestConfig = '';
+	for ( const item of result ) {
+		if ( item.config.length > longestConfig.length ) {
+			longestConfig = item.config;
+		}
+	}
+
+	return longestConfig.length;
```

Comment:
As an alternative approach, you could consider some slightly more functional code for this logic.
```suggestion
	const configLengths = result.map( ( item ) => item.config.length );
	return Math.max( ...configLengths );
```